src/app/components/client/ExposureCard.module.scss

```diff
@@ -30,20 +34,25 @@
     list-style: none;
     display: flex;
     align-items: center;
+    padding-left: 0;
     gap: $layout-md;
     margin: 0;
-    padding: 0 $spacing-sm;
     width: 100%;
+    justify-content: space-between;
 
     li {
-      flex: 1;
       display: flex;
       align-items: center;
-    }
 
-    .exposureImage {
-      max-width: 150px;
-      height: 100%;
+      @media screen and (min-width: $screen-md) {
+        flex: 1; 
+      }  
+      
+      .exposureImage {
+        width: 120px;
+        height: 100%;
+        object-fit: cover;
+      }
```

Comment:
Quick heads-up: AFAIK we haven't come up with a way to get proper rectangular company logos here, so this will probably need to become favicons at some point.

Reply:
You can see them next to the company names at https://monitor.firefox.com/breaches.